shuup/admin/templates/shuup/admin/macros/multilanguage.jinja

```diff
@@ -40,6 +40,9 @@ When using multiple language_dependent_content_tabs in a single page, do remembe
                 {% endif %}
             </div>
         {% endfor %}
+        <p class="text-center">
+            {% trans %}Partially translated languages does not fallback to default language{% endtrans %}
```

Comment:
Are we sure we want this message to always be visible? Maybe this is something that should be documented in the guide.
If we do then maybe use this: "Partially translated fields do not fall back to the default language"